modules/alb/versions.tf

```
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 5.17"
+      version = "> 5.17"

```

shouldn't we make it all like >= ?
```suggestion
      version = ">= 5.17"
```